src/vconsole/vconsole-setup.c

```diff
@@ -231,6 +231,18 @@ static int verify_vc_kbmode(int fd) {
         return IN_SET(curr_mode, K_XLATE, K_UNICODE) ? 0 : -EBUSY;
 }
 
+static int verify_vc_kdmode(int fd) {
```

Comment:
"kdmode" is not a term the console layer uses in its ioctl iface, is it? maybe don't introduce such cryptic new terms. just call this "verify_vc_display_mode()"?